framer/DOMEventManager.coffee

```diff
@@ -9,21 +9,14 @@ EventManagerIdCounter = 0
 class DOMEventManagerElement extends EventEmitter
 
 	constructor: (@element) ->
-		@_elementGestureManager = new GestureManagerElement(@element)
 
 	addListener: (eventName, listener, capture=false) ->	
 		super(eventName, listener)
-		if Events.isGestureEvent eventName
-			@_elementGestureManager.addEventListener(eventName, listener, capture)
-		else
-			@element.addEventListener(eventName, listener)
+		@element.addEventListener(eventName, listener, false)
```

Comment:
I know that this is not PR related, but why does the event need to be added to the DOM and also to EventEmitter? Is it to have the `listeners` and all the other handful EventEmitter functions?